cmd/create/userrole/cmd.go

```diff
@@ -0,0 +1,319 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package userrole
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/spf13/cobra"
+	errors "github.com/zgalor/weberr"
+
+	linkuser "github.com/openshift/rosa/cmd/link/userrole"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/aws/tags"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args struct {
+	prefix              string
+	permissionsBoundary string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "user-role",
+	Aliases: []string{"userrole"},
+	Short:   "Create user role to verify account association",
+	Long: "Create user role that allows OCM to verify that users creating a cluster " +
+		"have access to the current AWS account.",
+	Example: ` # Create user roles 
+  rosa create user-role --prefix `,
+	Run: run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	flags.StringVar(
+		&args.prefix,
+		"prefix",
+		"",
```

Comment:
In testing this, I think we should have the `aws.DefaultPrefix` prefix as the default. Makes everything easier and consistent